go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/rekor
 
-go 1.20
+go 1.21
```

Comment:
do we need to go to 1.21 for this?

Reply:
+1, let's not bump the Go version unless there's a 1.21 feature you're adding.

Reply:
I would use `x/exp` for now, I'd prefer to bump the go version in its own PR.